packages/wp-story-editor/src/components/documentPane/publish/publish.js

```diff
@@ -414,6 +447,41 @@ function PublishPanel({ nameOverride }) {
             </LabelWrapper>
           </DropdownWrapper>
         </HighlightRow>
+        {improvedAutosaves && (
+          <RevisionsWrapper>
+            <LabelWrapper>
+              <Label>
+                <LabelIconWrapper>
+                  <Icons.History width={24} height={24} aria-hidden />
+                </LabelIconWrapper>
+                <RevisionsLabel>
+                  {sprintf(
+                    /* translators: %d: number of revisions. */
+                    _n(
+                      '%d Revision',
+                      '%d Revisions',
+                      revisionCount,
+                      'web-stories'
+                    ),
+                    revisionCount
+                  )}
+                </RevisionsLabel>
+              </Label>
```

Comment:
Maybe we don't show this until we have at least one revision. It looks a little weird with 0. 

<img width="135" alt="Screenshot 2022-09-09 at 12 48 00" src="https://user-images.githubusercontent.com/237508/189343299-56fb81fa-f023-4199-9c94-df0e5c092607.png">

Reply:
Ya --- I had it hidden at one point --- will re-hide

Reply:
Styling nitpic, I think that manage link should be on the same line. Compare. 

<img width="172" alt="Screenshot 2022-09-09 at 13 06 00" src="https://user-images.githubusercontent.com/237508/189346322-69285315-2067-4872-9dc1-f3d3c1326e4c.png">
<img width="172" alt="Screenshot 2022-09-09 at 13 04 23" src="https://user-images.githubusercontent.com/237508/189346137-636b9514-d6d7-4c0f-ab0e-c31bfdf437ef.png">
<img width="141" alt="Screenshot 2022-09-09 at 13 04 19" src="https://user-images.githubusercontent.com/237508/189346140-3ff2e4c8-0ff8-42c0-89fa-455ff159a213.png">

Reply:
Moved to single line